topics/assembly/tutorials/vgp_genome_assembly/tutorial.md

```diff
@@ -824,7 +824,7 @@ The *Bionano Hybrid Scaffold* tool automates the scaffolding process, which incl
 
 > <hands-on-title>Bionano hybrid scaffolding</hands-on-title>
 >
-> 1. {% tool [Bionano Hybrid Scaffold](toolshed.g2.bx.psu.edu/repos/bgruening/bionano_scaffold/bionano_scaffold/3.6.1+galaxy2) %} with the following parameters:
+> 1. {% tool [Bionano Hybrid Scaffold](toolshed.g2.bx.psu.edu/repos/bgruening/bionano_scaffold/bionano_scaffold/3.6.1+galaxy2) %} (**VERSION 3.7+GALAXY0**) with the following parameters:
```

Comment:
Why this older version?

Reply:
ah, sorry -- when Guido was testing the tutorial he said the newer version didn't work, so I put this edit to use the older version that we have tested before, but he updated me and said that the newer version did work eventually, so we can use the newer version! i'll take this note out

Reply:
I see, we fixed that recently.